e2e/wdio/platform/tests/time-picker.e2e-spec.ts

```diff
@@ -93,35 +99,35 @@ describe('Time picker suite', function() {
         }
     });
 
-    xit('Verify null validity for basic time picker ', () => {
-        scrollIntoView(activeTimePickerButton, 4);
+    it('Verify null validity for basic time picker ', () => {
+        scrollIntoView(activeTimePickerButton, 5);
         expect(doesItExist(errorBorder)).toBe(false);
         click(setToNullButton);
         expect(doesItExist(errorBorder)).toBe(true);
         click(setValidTimeButton);
         expect(doesItExist(errorBorder)).toBe(false);
-        expect(getValue(activeTimePickerInput, 4)).toBe(defaultValidTime);
+        expect(getValue(activeTimePickerInput, 5)).toBe(defaultValidTime);
     });
 
     // skipped due to https://github.com/SAP/fundamental-ngx/issues/4853
     xit('Verify null validity for time picker with reactive form', () => {
-        scrollIntoView(activeTimePickerButton, 9);
+        scrollIntoView(activeTimePickerButton, 10);
```

Comment:
i see the test has been modified but it is still excluded as `xit`. Is this intentional?

Reply:
This was intentional. I updated the selector for activetimePickerButton and knew the index here needed to be updated. That being said, I saw the issue that was blocking this test case has been fixed, so I went ahead and updated this case now.

Reply:
>That being said, I saw the issue that was blocking this test case has been fixed, so I went ahead and updated this case now.
I am a bit confused, so is this test supposed to be `it` now that you updated the testcase or should it still be `xit`?

Reply:
oh something was wrong with my github i guess, it was showing your old code, please ignore my comment